README.md

```diff
@@ -11,52 +11,76 @@ in ScalaDoc.
 To use this plugin, add it to your `project/plugins.sbt`.
 
 ```scala
-addSbtPlugin("com.github.tkawachi" % "sbt-doctest" % "0.5.0")
+addSbtPlugin("com.github.tkawachi" % "sbt-doctest" % "0.5.1")
 ```
 
 It's automatically enabled for JVM projects.
 Scala.js is currently not supported (See #52).
 
+### Using ScalaTest
 
-### Using ScalaTest or Specs2
-
-This plugin generates tests for ScalaCheck by default. If you use ScalaTest or Specs2,
-set `doctestTestFramework` to `DoctestTestFramework.ScalaTest` or `DoctestTestFramework.Specs2` in `build.sbt`.
-Then it will generate tests for the specified framework.
+If you are using ``ScalaTest``, add the following lines to your ``build.sbt``:
 
 ```scala
-// To generate tests for ScalaTest
+libraryDependencies ++= Seq(
+  "org.scalatest" %% "scalatest" % "3.0.1" % "test"
```

Comment:
Please add a scalacheck dependency corresponding to scalatest for property testing.